<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvOptDialog</class>
 <widget class="QWidget" name="ConvOptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lbRinexVersion">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>RINEX Version</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cBRinexVersion">
        <property name="toolTip">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>2.10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2.11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2.12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.01</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.02</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.03</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.04</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cBSeperateNavigation">
        <property name="text">
         <string>Sep NAV</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbStationID">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Station ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lERinexStationCode">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>0000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbTimeTolerance">
        <property name="text">
         <string>Time Torelance (s)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="sBTimeTolerance">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
        <property name="value">
         <double>0.005000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cBRinexFilename">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>RINEX2 Name</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox1">
     <property name="toolTip">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="9" column="0" colspan="7">
       <widget class="QWidget" name="options" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="cBPhaseShift">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Phase Shift</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cBHalfCycle">
           <property name="text">
            <string>Half Cyc Corr</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cBOutputIonoCorr">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Iono Corr</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cBOutputTimeCorr">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Time Corr</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cBOutputLeapSecs">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Leap Sec</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="5" colspan="2">
       <widget class="QLineEdit" name="lEMarkerType">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbRunBy">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>RunBy/Obsrv/Agency</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="cBAutoPosition">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lbApproxPosition">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Approx Pos XYZ</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="lEMarker">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="2">
       <widget class="QLineEdit" name="lEObserver">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lbAntennaDelta">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Antenna Delta H/E/N</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5" colspan="2">
       <widget class="QLineEdit" name="lEReceiverVersion">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="5" colspan="2">
       <widget class="QLineEdit" name="lEAgency">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lbAntenna">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Ant #/Type</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="5">
       <widget class="QLineEdit" name="lEComment1">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="lEAntenna">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lbReceiver">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Rec #/Type/Vers</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lERunBy">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="3" colspan="2">
       <widget class="QLineEdit" name="lEReceiverType">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbMarker">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Maker Name/#/Type</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3" colspan="2">
       <widget class="QLineEdit" name="lEAntennaType">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="5">
       <widget class="QLineEdit" name="lEComment0">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="lEReceiver">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2">
       <widget class="QLabel" name="lbComment">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Comment</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" colspan="2">
       <widget class="QLineEdit" name="lEMarkerNo">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QDoubleSpinBox" name="sBApproxPosition0">
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="3" colspan="2">
       <widget class="QDoubleSpinBox" name="sBApproxPosition1">
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="5" colspan="2">
       <widget class="QDoubleSpinBox" name="sBApproxPosition2">
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QDoubleSpinBox" name="sBAntennaDelta0">
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="8" column="5" colspan="2">
       <widget class="QDoubleSpinBox" name="sBAntennaDelta2">
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="8" column="3" colspan="2">
       <widget class="QDoubleSpinBox" name="sBAntennaDelta1">
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lEExcludedSatellites">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="lbExcludedSatellites">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Excluded Satellites</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="2">
       <widget class="QGroupBox" name="groupBox2">
        <property name="toolTip">
         <string/>
        </property>
        <property name="title">
         <string>Satellite Systems</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="cBNavSys1">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>GPS</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cBNavSys2">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>GLO</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cBNavSys3">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>GAL</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cBNavSys4">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>QZS</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cBNavSys6">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>BDS</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cBNavSys7">
           <property name="text">
            <string>NavIC</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cBNavSys5">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>SBS</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox3">
        <property name="toolTip">
         <string/>
        </property>
        <property name="title">
         <string>Observation Types</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="cBObservationTypeC">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>C</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cBObservationTypeL">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>L</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cBObservationTypeD">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>D</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cBObservationTypeS">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>S</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox4">
        <property name="toolTip">
         <string/>
        </property>
        <property name="title">
         <string>GNSS Signals</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="cBFreq1">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>L1</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cBFreq2">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>L2</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cBFreq3">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>L3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cBFreq4">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>L4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cBFreq5">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>L5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMask">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Mask...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnFcn">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>FCN...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnFrequencies">
           <property name="text">
            <string>?</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lBReceiverOptions">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Receiver Options</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lEReceiverOptions">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbDebug">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Debug</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cBTraceLevel">
        <property name="toolTip">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>OFF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Level 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Level 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Level 3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Level 4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Level 5</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnOk">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;OK</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>groupBox1</zorder>
  <zorder>widget</zorder>
  <zorder>widget_2</zorder>
  <zorder>widget_3</zorder>
  <zorder>widget_4</zorder>
 </widget>
 <tabstops>
  <tabstop>cBRinexVersion</tabstop>
  <tabstop>lERinexStationCode</tabstop>
  <tabstop>cBRinexFilename</tabstop>
  <tabstop>lERunBy</tabstop>
  <tabstop>lEObserver</tabstop>
  <tabstop>lEAgency</tabstop>
  <tabstop>lEComment0</tabstop>
  <tabstop>lEComment1</tabstop>
  <tabstop>lEMarker</tabstop>
  <tabstop>lEMarkerNo</tabstop>
  <tabstop>lEMarkerType</tabstop>
  <tabstop>lEReceiver</tabstop>
  <tabstop>lEReceiverType</tabstop>
  <tabstop>lEReceiverVersion</tabstop>
  <tabstop>lEAntenna</tabstop>
  <tabstop>lEAntennaType</tabstop>
  <tabstop>cBAutoPosition</tabstop>
  <tabstop>sBApproxPosition0</tabstop>
  <tabstop>sBApproxPosition1</tabstop>
  <tabstop>sBApproxPosition2</tabstop>
  <tabstop>sBAntennaDelta0</tabstop>
  <tabstop>sBAntennaDelta1</tabstop>
  <tabstop>sBAntennaDelta2</tabstop>
  <tabstop>cBPhaseShift</tabstop>
  <tabstop>cBOutputIonoCorr</tabstop>
  <tabstop>cBNavSys1</tabstop>
  <tabstop>cBNavSys2</tabstop>
  <tabstop>cBNavSys3</tabstop>
  <tabstop>cBNavSys4</tabstop>
  <tabstop>cBNavSys6</tabstop>
  <tabstop>cBNavSys7</tabstop>
  <tabstop>lEExcludedSatellites</tabstop>
  <tabstop>cBObservationTypeC</tabstop>
  <tabstop>cBObservationTypeL</tabstop>
  <tabstop>cBObservationTypeD</tabstop>
  <tabstop>cBObservationTypeS</tabstop>
  <tabstop>cBFreq1</tabstop>
  <tabstop>cBFreq2</tabstop>
  <tabstop>cBFreq3</tabstop>
  <tabstop>cBFreq4</tabstop>
  <tabstop>cBFreq5</tabstop>
  <tabstop>btnMask</tabstop>
  <tabstop>lEReceiverOptions</tabstop>
  <tabstop>cBTraceLevel</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
